### ISSUE TITLE
Current type-checker of redtt is not happy with strict bools and nats

### ISSUE THREAD

**Issue by favonia** [RedPRL/redtt#157]
@jonsterling @cangiuli You can probably prove equality reflection for strict bools and nats through hcom, so the type checker is probably irritated when I use hcom of nat.

**Comment by jonsterling** [comment]
Can you clarify the issue? I think we cant use the naive operational semantics for kan operations in Nat, but I'm not sure.

**Comment by favonia** [comment]
We probably have to treat those rigid hcoms in nat/bool as values.

**Comment by jonsterling** [comment]
I don't think that makes any sense... Can we check what they do in cubicaltt/yacctt? What seems much more likely is that we have neutral hcoms.

**Comment by favonia** [comment]
@mortberg @cangiuli How strict the natural numbers are in cubicaltt/yacctt?
